### PR TITLE
AllegroCL event class fix

### PR DESCRIPTION
**Pull request by dg1sbg**
Separated event.lisp into two files. Metaclass definition now in sepa…rate file. AllegroCL needs meta-class defined before loading/compiling event class.

### CODE REVIEW

File: event-class.lisp [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#|
+ This file is a part of deeds
+ (c) 2015 Shirakumo http://tymoon.eu (shinmera@tymoon.eu)
+ Author: Nicolas Hafner <shinmera@tymoon.eu>
+|#
+
+(in-package #:org.shirakumo.deeds)
+
+;;; eval-when workaround for LW
+(eval-when (:compile-toplevel :load-toplevel :execute)
```
Comment (Shinmera) [inline event-class.lisp]: Since it's in a separate file the eval-when should be removed now.

File: event.lisp [hunk 2]
```diff
@@ -6,81 +6,14 @@
...
-
 (defclass event ()
   ((origin :initarg :origin :reader origin :mutable NIL)
    (event-loop :initform NIL :reader event-loop :mutable NIL)
    (issue-time :initform NIL :accessor issue-time :mutable T)
    (cancelled :initform NIL :accessor cancelled :mutable T))
   (:metaclass event-class)
   (:default-initargs
-   :origin *origin*))
+      :origin *origin*))
```
Comment (Shinmera) [inline event.lisp]: Please revert this accidental indentation change.

**Comment by Shinmera** [comment]
Thanks!

**Comment by dg1sbg** [comment]
+1 - Thanks!

**Pull request merged**